Update how to run your own daprd.exe images (#1962)
dapr/docs: daprdocs/content/en/reference/arguments-annotations-overview.md

| `--dapr-grpc-port` | `--dapr-grpc-port` | | not supported | gRPC port for the Dapr API to listen on (default "50001") |
| `--dapr-http-port` | `--dapr-http-port` | | not supported | The HTTP port for the Dapr API |
|` --dapr-http-max-request-size` | --dapr-http-max-request-size | | `dapr.io/http-max-request-size` | Increasing max size of request body http and grpc servers parameter in MB to handle uploading of big files. Default is `4` MB |

> | not supported | `--image` | | `dapr.io/sidecar-image` | Dapr sidecar image. Default is daprio/daprd:latest. One scenario this can be used is when building your own custom image of `darpd.exe` and you want this launched by the Kubernetes Sidecar Injector service rather than the latest supported version |

Reviewer: ```suggestion
| not supported | `--image` | | `dapr.io/sidecar-image` | Dapr sidecar image. Default is daprio/daprd:latest. Use this when building your own custom image of Dapr and the Kubernetes Sidecar Injector will use this image instead of the default version of Dapr |
```
